src/mcp-bundle/tests/DependencyInjection/McpBundleTest.php

```diff
@@ -49,6 +50,10 @@ public function testCustomConfiguration()
 
     public function testMcpLoggerServiceIsCreated()
     {
+        if (!class_exists(MonologBundle::class)) {
+            $this->markTestSkipped('MonologBundle is not installed');
+        }
+
```

Comment:
Since monolog is now installe in dev we don't need this ?

Reply:
b920d78289481b3d1594f91063858f8f1d4b055a